plugins/chat/app/services/chat/update_message.rb

```diff
@@ -98,6 +100,8 @@ def save_message(message:, **)
     end
 
     def save_revision(message:, guardian:, **)
+      return false if message.streaming_before_last_save
```

Comment:
I can't find where `streaming_before_last_save` is defined, maybe this should just be ` if message.streaming`?